docs/source/services/timelock_service/product-changes.rst

```diff
@@ -0,0 +1,20 @@
+.. _product-changes:
+
+Developing a product to use the timelock service
+================================================
+ .. note::
+    We recommend only product instances deployed against Cassandra KVS use the timelock server, as we don't have automated migrations for other KVSs.
```

Comment:
I think we should probably just remove this box. I would be completely fine if a new customer with postgres/oracle wanted to install their service and use the timelock server. We already have multiple warnings that automatic migrations only exist for cassandra, so I don't think we need this.